static/style.css

```diff
@@ -73,3 +78,32 @@ iframe {
     }
   }
 }
+
+.callout {
+  padding: 10px;
+  margin: 20px 0;
+  border: 1px solid #eee;
+  border-left-width: 5px;
+  border-radius: 3px;
+  h4 {
+    margin-top: 0;
+    margin-bottom: 5px;
+  }
+  p:last-child {
+    margin-bottom: 0;
+  }
+  code {
+    border-radius: 3px;
+  }
+  & + .bs-callout {
+    margin-top: -5px;
+  }
+}
+@each $name,$color in (default,#777),(primary,#428bca),(success,#5cb85c),(danger,#d9534f),(warning,#f0ad4e),(info,#5bc0de), (bdc,#29527a) {
```

Comment:
Fungerer dette? Jeg tror egentlig dette er ment å brukes i en CSS-preprosesserer og ikke vil fungere i en .css-fil direkte

Reply:
Vi har løst dette på en annen måte. Takk igjen for ditt massive bidrafg.